src/test/java/com/microsoft/azure/spring/data/cosmosdb/core/DocumentDbTemplateIT.java

```diff
@@ -262,21 +265,21 @@ public void testPaginationAsync() {
     @Test
     public void testInsertAsync() {
         this.dbTemplate.deleteAll(personInfo.getCollectionName(), personInfo.getPartitionKeyNames());
-        this.dbTemplate.insertAsync(this.personInfo.getCollectionName(), TEST_PERSON_0, null)
-                .subscribe(p -> assertThat(p).isEqualTo(TEST_PERSON_0));
-        this.dbTemplate.insertAsync(this.personInfo.getCollectionName(), TEST_PERSON_1, null)
-                .subscribe(p -> assertThat(p).isEqualTo(TEST_PERSON_1));
+        Person inserted = this.dbTemplate.insertAsync(this.personInfo.getCollectionName(), TEST_PERSON_0, null)
```

Comment:
Save  `this.personInfo.getCollectionName()` to a variable for reuse?

Reply:
make sense, will cleanup this in another PR.